components/payment/AccountTabContent.tsx

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import { View, Image, StyleSheet, FlatList } from "react-native";
+
+import BankAccountDetail from "./BankAccountDetail";
+import PrimaryButton from "../button/PrimaryButton";
+import SubHeaderText from "../text/SubHeaderText";
+
+import Colors from "@/constants/color";
+import { MOCKED_BANK_ACCOUNT, mockedBankAccount } from "@/mock/mockData";
+
+type AccountTabProps = {
+  name?: string;
+};
+
+const IMAGE_SIZE = 100;
+
+const AccountTabContent: React.FC<AccountTabProps> = () => {
+  return (
+    <View style={styles.contentContainer}>
+      <View style={styles.profileContainer}>
+        <Image
+          style={styles.profileImage}
+          source={{
+            uri: "https://fastly.picsum.photos/id/157/200/300.jpg?hmac=-OZWQAIRoAdYWp7-qnHO1wl5t0TO3BMoAgW3tmR7wgE",
+          }}
+          height={IMAGE_SIZE}
+          width={IMAGE_SIZE}
+        />
+        <SubHeaderText
+          text="Worashot Chanangkarn"
+          textStyle={{ color: Colors.blue[500] }}
+        />
+      </View>
+      <View>
+        <FlatList
+          data={MOCKED_BANK_ACCOUNT}
+          keyExtractor={(item: mockedBankAccount) => item.id}
+          renderItem={({ item }) => (
+            <BankAccountDetail
+              accountNo={item.id}
+              bankAccountNumber={item.bankAccountNumber}
+              bankName={item.bankName}
+              bankBranch={item.bankBranch}
+            />
+          )}
+          overScrollMode="never"
+        />
+      </View>
+      <PrimaryButton
+        title="Add Account"
+        onPress={() => {}}
+        buttonStyle={styles.button}
+        textStyle={styles.textButton}
+      />
+    </View>
+  );
+};
+
+export default AccountTabContent;
+
+const styles = StyleSheet.create({
+  contentContainer: {
+    flex: 1,
+    backgroundColor: Colors.gray[50],
+  },
```

Comment:
Can we move this part to be the style of children in `TabsContainer.tsx`? since I see this kind of things every tab content

Reply:
`TabsContainer.tsx` receive only the content so it don't wrap them with other tags. But I can move them into that file if you want?

Reply:
fixed on [5a0ad41](https://github.com/ParkingSystemWithLPR/Napark-application/pull/28/commits/5a0ad41ad0f832d534d1b597892bf326a7969713)